workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/ReaderWriterFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Improbable.Gdk.Core
+{
+    public static class ReaderWriterFactory
+    {
+        public static object CreateReaderWriter(uint componentId)
+        {
+            throw new NotImplementedException();
```

Comment:
Wondering if we should target an experimental branch for `monobehaviour stuff` rather than `master` if we're going to be merging in `NotImplementedException`s. Thoughts?

Reply:
As long as we're not making anything nonfunctional actually run I'd say it's okay, especially since it shouldn't take a massively long time? And even though other changes in master shouldn't affect us much, when they do it's nice to keep up with them and not have huge merge conflicts at the end.

Reply:
It's still good to prevent committing partial functionality into master IMO, it should be trivial to retarget onto the branch

Reply:
Let's have the feature branch regardless just to follow a clean approach. It is indeed weird to have code that throws `NotImplementedExceptions` checked into master. Same for the placeholder unit tests. Master should be a shining oasis sort of thing. ;)

Reply:
+1. Don't merge this into master. However make it an experimental branch as it needs to have PRs to add code to it.